pom.xml

```diff
@@ -42,7 +42,7 @@
          "version.spring.cloud.azure" is used to update "_configprops.adoc".
```

Comment:
do we need to update this comments here?

Reply:
deleted